src/styles/print.css

```diff
@@ -41,7 +41,8 @@
     /* Adding this border makes it work in FF 39 on Linux. I have no
        Idea why, but please don't remove it !! */
     border: 1px solid #ddd;
-    page-break-after: always;
```

Comment:
[HFP-3633 comment about blink redering engine](https://h5ptechnology.atlassian.net/browse/HFP-3663?focusedCommentId=41669) 
Seem to be a very weird thing happening with Blink rendering engine and page-break properties. page-break-inside worked just fine, not really sure why.